docusign/1.0/modules/ROOT/pages/docusign-connector-studio.adoc

```diff
@@ -0,0 +1,184 @@
+
+To configure the global element for DocuSign Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, in *Connection*, select the authentication method to configure:
+
+* <<auth-connection, Authorization Connection Provider (Implicit Grant)>>
+
+* <<oauth-connection, OAuth Connection Provider (Authorization Code Grant)>>
+
++
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, refer to xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+
+[[auth-connection]]
+=== Authorization Connection Provider (Implicit Grant)
+
+. Enter the following information on the *General* tab of the *Global Element Properties* screen to configure Authorization Connection Provider (Implicit Grant):
+
++
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Name |Enter the configuration name.
+|Base Uri | Parameter base URI for each instance or tenant.
+|Authorization | Authorization for the connection type.
+|===
++
+
++
+The following screenshot shows an example of configuring the *General* tab for Authorization Connection Provider (Implicit Grant):
++
+
++
+image::auth-general-tab.png[To configure Authorization Connection Provider (Implicit Grant) authentication, select `Authorization Connection Provider (Implicit Grant)` in the *Connection* field and then complete the fields on the *General* tab.]
++
+
+. On the *Advanced* tab, optionally change the default values of the *Response timeout*, *Response timeout unit*, *Streaming type*, and *Expiration policy* fields.
+
++
+The following screenshot shows an example of configuring the *Advanced* tab for Authorization Connection Provider (Implicit Grant):
++
+
++
+image::auth-advanced-tab.png[To configure Authorization Connection Provider (Implicit Grant) authentication, select `Authorization Connection Provider (Implicit Grant)` in the *Connection* field and then optionally complete the fields on the *Advanced* tab.]
```

Comment:
This caption is too long. The info in it was already presented in the previous steps.